index.d.ts

```diff
@@ -32,6 +32,11 @@ export interface Options {
 	@default true
 	*/
 	readonly waitForChildren?: boolean;
+
+	/**
```

Comment:
This needs a description (including mentioning its use-cases), and ideally a realistic code example.